service/scaladsl/server/src/main/scala/com/lightbend/lagom/scaladsl/server/LagomApplicationLoader.scala

```diff
@@ -5,6 +5,7 @@ package com.lightbend.lagom.scaladsl.server
 
 import java.net.URI
 
+import akka.actor.CoordinatedShutdown.JvmExitReason
```

Comment:
Unused import

Reply:
Yep. I initially thought about using `JvmExitReason`, but obviously this is not the right reason. That’s why I introduced the `ApplicationShutdownReason` like Play is doing.